src/Raven.Client/ServerWide/Operations/Certificates/EditClientCertificateOperation.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations.Certificates
+{
+    public class EditClientCertificateOperation : IServerOperation
+    {
+        private readonly string _thumbprint;
+        private readonly Dictionary<string, DatabaseAccess> _permissions;
+        private readonly string _name;
+        private readonly SecurityClearance _clearance;
+
+        public EditClientCertificateOperation(string thumbprint, string name, Dictionary<string, DatabaseAccess> permissions, SecurityClearance clearance)
```

Comment:
introduce nested `Parameters` class (we do not that in other places) and use those. Easier to extend in the future.